README.md

```diff
@@ -19,6 +19,7 @@ composer create-project govcms/govcms8-scaffold-paas MY_PROJECT
  * [Drupal/GovCMS distribution](https://govcms.gov.au/wiki-distro)
  * [GovCMS Platform](https://govcms.gov.au/wiki-platform)
  * [GovCMS Maintenance](https://govcms.gov.au/wiki-maintenance)
+ * [GovCMS release process](https://github.com/govCMS/govcms8-scaffold-paas/wiki/Update-process)
```

Comment:
Please use a redirect in govcms.gov.au so it's easier to maintain. This link could change for multiple reasons. There is no control over the URL in the wiki so if the title was changed the URL will change. That is the only page in that wiki. It's arguably an orphan wiki that would be consolidated if there was ever a cohesive approach to the documentation for govcms. So yeah, a redirect in govcms is a nice protection against this.

Reply:
Thanks @simesy! I've updated to reference a redirect, will follow up with the team to get the redirect added to govcms.gov.au.